wallet/chain/p/builder/builder.go

```diff
@@ -677,6 +713,33 @@ func (b *builder) NewImportTx(
 		})
 	}
 
+	var (
+		toBurn     map[ids.ID]uint64
```

Comment:
This section can be simplified further.

```go
toBurn := map[ids.ID]uint64{}
toStake := map[ids.ID]uint64{}
var excessAVAX uint64

if avax := importedAmounts[avaxAssetID]; avax > txFee {
    excessAVAX = avax - txFee
} else {
    toBurn[avaxAssetID] = txFee - avax
}
```

Reply:
Sorry, your [explanation](https://github.com/ava-labs/avalanchego/pull/3282#discussion_r1711870623) made me realise that it should be `avax >= txFee`.

Reply:
I think the code actually works regardless of `avax >= txFee` or `avax > txFee` (as setting `toBurn[avaxAssetID] = 0` is fine... Although slightly inefficient I suppose.

Reply:
Agreed, it should be functionally identical. Just feels "more correct" to leave `excessAVAX == 0` instead of also adding a zero burn.